include/drivers/console/native_posix_console.h

```diff
@@ -2,6 +2,7 @@
  * Copyright (c) 2018 Oticon A/S
  *
  * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileComment: IEC-61508-SIL3
```

Comment:
Also seems like a false positive

Reply:
Very likely mistakenly marked.

Reply:
Please see 

## Concerning this PR and the 800+ files changed

in the description.